src/app/service/categories/ServiceCategories.jsx

```diff
@@ -61,26 +69,32 @@ class ServiceCategories extends Component {
             <Header>What programs and services are available at this location?</Header>
           </div>
           <Accordion>
-            <Accordion.Item
-              active={active === 0}
-              onClick={() => this.onToggleOpen(0)}
-              title="All Services"
-              icon="home"
-            />
-            <Accordion.Content active={active === 0}>
-              <Selector fluid>
-                {taxonomy.map(category => (
-                  <Selector.Option
-                    key={category.id}
-                    onClick={() => this.onSelect(category)}
-                    active={selected[category.id]}
-                  >
-                    {category.name}
-                  </Selector.Option>
-                ))}
-                <Selector.Option align="center">+ Add another shelter service</Selector.Option>
-              </Selector>
-            </Accordion.Content>
+            {taxonomy.map((category, i) => (
```

Comment:
Probably makes more sense for active to be based on the category ID, rather than the index, no?